.github/ISSUE_TEMPLATE/troubleshooting.md

```diff
@@ -0,0 +1,10 @@
+<!-- ⚠️⚠️STOP⚠️⚠️ -- PLEASE READ! -->
+
+We use GitHub to track feature requests and bug reports. Please **do not** submit issues
+for questions about how to configure, use features, troubleshoot, or best practices for
+using New Relic software.
+
+See the README.md support section in this repository for more details on
```

Comment:
Pointed this at known to exist "support" section instead of "troubleshooting" section that does not exist.